index.html

```diff
@@ -17,16 +17,6 @@ <h2>Devslopes</h2>
       <h3>Learn to Code</h3>
     </div>
```

Comment:
did we need to remove the css for that glowing dots, since we removed them from the html

Reply:
oh youre right, thanks for pointing that out.